dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/loadbalance/ConsistentHashLoadBalance.java

```diff
@@ -72,6 +84,28 @@
 
         private final int[] argumentIndex;
 
+        /**
+         * key: server(invoker) address
+         * value: count of requests accept by certain server
+         */
+        private Map<String, Long> serverRequestCountMap = new HashMap<>();

```

Comment:
Change to ConcurrentHashMap to prevent concurrent problem

Reply:
I have resolved this problem.